README.md

```diff
@@ -771,6 +775,119 @@ Known Issues
 
 Due to [a change in ActiveRecord's `count` method](https://github.com/rails/rails/pull/10710) you will need to use `count(:all)` to explicitly count all columns ("*") when using a `near` scope. Using `near` and calling `count` with no argument will cause exceptions in many cases.
 
+
+#### Using `near` with `select`
+
+You cannot reduce number of getting columns using [ActiveRecord's `select` method](https://api.rubyonrails.org/classes/ActiveRecord/QueryMethods.html#method-i-select) after `near` scope.
+
+For example, the following code:
+
+```ruby
+Venue.near("Austin, TX", 200, min_radius: 40).select(:id, :address)
+```
+
+will produce a SQL as follows:
+
+```sql
+SELECT
+    venues.*, -- unexpectedly added
+    (69.09332411348201 * ABS(venues.latitude - 30.2711286) * 0.7071067811865475) + (59.836573914187355 * ABS(venues.longitude - - 97.7436995) * 0.7071067811865475) AS distance,
+    CASE
+        WHEN(
+            venues.latitude >= 30.2711286
+        AND venues.longitude >= - 97.7436995
+        ) THEN 45.0
+        WHEN(
+            venues.latitude < 30.2711286
+        AND venues.longitude >= - 97.7436995
+        ) THEN 135.0
+        WHEN(
+            venues.latitude < 30.2711286
+        AND venues.longitude < - 97.7436995
+        ) THEN 225.0
+        WHEN(
+            venues.latitude >= 30.2711286
+        AND venues.longitude < - 97.7436995
+        ) THEN 315.0
+    END AS bearing,
+    "venues"."id",     -- added
+    "venues"."address" -- added
+FROM
+    "venues"
+WHERE
+    (
+        venues.latitude BETWEEN 27.37649293778304 AND 33.16576426221696
+    AND venues.longitude BETWEEN - 101.09533124708325 AND - 94.39206775291676
+    )
+ORDER BY
+    distance ASC
+```
+
+because `venues.*` added by `near` scope is preferred to columns you specified.
+This may cause performance degradation.
+
+Instead, you need to use `near` scope with [`select` option](https://github.com/alexreisner/geocoder/blob/b546dd637672b9bf84abf692b4594f39cdc0ad48/lib/geocoder/stores/active_record.rb#L104) in such situation.
+
+For example, the following code:
+
+```ruby
+Venue.near("Austin, TX", 200, min_radius: 40, select: "id, name")
+```
+
+will produce a SQL as follows:
+
+```sql
+SELECT
+    id,
+    address,
+    (69.09332411348201 * ABS(venues.latitude - 30.2711286) * 0.7071067811865475) + (59.836573914187355 * ABS(venues.longitude - - 97.7436995) * 0.7071067811865475) AS distance,
+    CASE
+        ...
+    END AS bearing
+FROM
+    "venues"
+...
+```
+
+The above SQL has only necessary columns.
+
+#### Using `near` with `pluck`
+
+You cannot use [ActiveRecord's pluck method](https://api.rubyonrails.org/classes/ActiveRecord/Calculations.html#method-i-pluck) after `near` scope.
+
+For example, the following code:
+
+```ruby
+
```

Comment:
```suggestion
```

Reply:
@kou 
Thanks for your review!
I will fix your suggestion.

Reply:
Fixed in commit: https://github.com/alexreisner/geocoder/pull/1639/commits/b1debc0eb50e3224a834a68ad4d14041537acb97